tools-local/tasks/mobile.tasks/WasmBundleTask/WasmBundleTask.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <OutputType>Library</OutputType>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
```

Comment:
Why is this needed? If this can be dropped, then the `<Compile..` items can be removed too.